docs/providers/aws/guide/serverless.yml.md

```diff
@@ -285,6 +285,11 @@ provider:
       format: '{ "requestId":"$context.requestId", "ip": "$context.identity.sourceIp", "requestTime":"$context.requestTime", "httpMethod":"$context.httpMethod","routeKey":"$context.routeKey", "status":"$context.status","protocol":"$context.protocol", "responseLength":"$context.responseLength" }'
 
     frameworkLambda: true # Optional, whether to write CloudWatch logs for custom resource lambdas as added by the framework
+  s3: # If you need to configure the bucket itself, you'll need to add s3 resources to the provider configuration: 
+    bucketOne: # Eventual additional properties in camel case
```

Comment:
I believe it is worth mentioning that properties that are supported are the same ones as supported by CF resource for S3 bucket: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket.html

Reply:
Great suggestion. I have updated the docs.